src/main/java/com/zaiats/catcher/model/CandidateModel.java

```diff
@@ -2,14 +2,27 @@
 
 import com.zaiats.catcher.repository.entity.Candidate;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
 import java.util.Objects;
 
 public class CandidateModel {
 
     private Long id;
+
+    @NotBlank
+    @Size(max = 64)
     private String firstName;
+
+    @NotBlank
+    @Size(max = 64)
     private String lastName;
+
+    @Email
     private String email;
```

Comment:
Add message, when email will not valid return message.
`@Email(message = "{email.not.valid}")`

Reply:
1. Done!
2. Good catch ^^)
Done.